### PR TITLE
Feature/light weight cal experiment

### PR DESCRIPTION
**Pull request by davideastmond**
[Phase 1]
- Calendar setup
- Allow creation of occurrences (events)
- Sends data to back-end correctly formatted
- Implements google maps address autocomplete - note you'll need to enable this in your cloud console and get your own API key

### CODE REVIEW

File: src/app/adventure/view/[adventureId]/page.tsx [hunk 1]
```diff
@@ -1,16 +1,56 @@
...
+      title: "Direct Object",
+      start: dayjs().hour(4).minute(0).toDate(),
+      end: dayjs().hour(5).minute(0).toDate(),
+    },
+    {
+      _id: "4",
+      title: "Test2",
+      start: dayjs().add(1, "day").hour(4).minute(0).toDate(),
+      end: dayjs().add(1, "day").hour(5).minute(0).toDate(),
+    },
```
Comment (davideastmond) [inline src/app/adventure/view/[adventureId]/page.tsx:46]: Test data. It will be deleted

**Comment by varchodi** [comment]
LGTM

**Review by varchodi — APPROVED** [review]
LGTM 🫡;

**Review by BrianStas — APPROVED** [review]
Code is working as expected. Testing the features, I was able to access the calendar, drag a box onto the calendar to indicate times and then input information according to the type of booking selected at top. This information was then accessible in MongoDB under the adventure collection. This information, though not yet displayed, is built as expected and therefore this request has been approved.

**Pull request merged**